pages/governance.md

```diff
@@ -0,0 +1,235 @@
+* [Oscar Dowson](https://github.com/odow)
+* [Changhyun Kwon (U. South Florida)](https://github.com/chkwon)
+
+If a member of the Steering Council leaves the Steering Council, the remaining
+members, in consultation with the core contributors, will invite a member of the
+community to join in order to maintain a quorum of five members.
+
+#### Conflict of Interest
+
+It is expected that Steering Council members will be employed at a wide range of
+companies, universities and non-profit organizations. Because of this, it is
+possible that Members will have conflicts of interest. Such conflicts of
+interest include, but are not limited to:
+
+* Financial interests, such as investments, employment or contracting work,
+  outside of JuMP that may influence their work on JuMP.
+* Access to proprietary information of their employer that could potentially
+  leak into their work with JuMP.
+
+All members of the Steering Council shall disclose to the Steering Council any
+conflict of interest they may have. Members with a conflict of interest in a
+particular issue may participate in Steering Council discussions on that issue,
+but must recuse themselves from authorizing or approving any expenditures.
+
+## Decision Making Process
+
+This section outlines how fiscal and non-fiscal decisions are made in the JuMP
+project.
+
+### Fiscal Decisions
+
+All fiscal decisions are made by the Steering Council to ensure any funds are
+spent in a manner that furthers the mission of JuMP. Fiscal decisions require
+majority approval by acting Steering Council members.
+
+The Steering Council can gain addition decision-making power if the core
+contributors decide to delegate, e.g., choosing a chair for a JuMP-dev workshop.
+
+Community members proposing decisions with a fiscal aspect should contact the
+Steering Council directly, or table their proposal as an agenda item for
+discussion on a Monthly developer call (see below).
+
+### Non-fiscal Decisions
+
+All non-fiscal decisions are made via consensus of the core contributors and
+relevant repository maintainers. Emeritus core contributors are not considered
+to be core contributors for this purpose.
+
+Code-related decisions, such when a pull request is ready to be accepted and
+merged, should be discussed via the relevant Github issues and pull requests. If
+consensus cannot be achieved, the community member proposing the change may be
+invited by a core contributor to present their proposal at a Monthly developer
+call for further discussion and community input.
+
+Non-code-related technical decisions, such long-term strategic planning for
+JuMP, should either be discussed in a Github issue, or tabled as an agenda item
+and discussed on a Monthly developer call.
+
+If consensus on a non-fiscal decision cannot be achieved, the final decision
+will be made by the BDFL.
+
+## Github Permissions
+
+For security, all core contributors and repository maintainers must have
```

Comment:
This is a setting that's enforced by GitHub, not sure how important it is to mention here. I would say that GitHub permissions are used to control access to repositories.

Also: Anyone with commit access to a repository is trusted to use it in a way that's consistent with the "non-fiscal decisions" process. Prefer pull requests over direct pushes, ask for feedback on changes if you're not sure there's a consensus, and follow JuMP's style guide and development processes.